### PR TITLE
Generate logs from requests

### LINKED ISSUE #3: Generate Logs
Currently, the request is sent using `scrips/request_curl.py` file which is set in the environment path inside the docker container as written [here](https://github.com/coding-blocks/poker/blob/master/Dockerfile#L15). 

Edit the `request_curl.py` to generate the logs after sending the request containing the status code and response from the server. To access the models inside the request_curl, Django's settings.py must be instantiated. You can refer [this](https://docs.djangoproject.com/en/3.0/howto/custom-management-commands/) link to create a custom Django admin command and execute it.

Since the crontab runs all the command from the home(~) directory of the user you'll need to Change Directory before executing the script [here](https://github.com/coding-blocks/poker/blob/master/services/crontab.py#L12)

### PR DESCRIPTION
**Pull request by prateek2211**
fix #3

### CODE REVIEW

**Comment by prateek2211** [comment]
A WIP, just need to uncomment a line, once discussed about job_id

File: cron/management/commands/sendrequest.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    def handle(self, *args, **options):
+        headers = {
+            'Authorization': f'Bearer {options["token"]}'
+        }
+
+        session = requests.Session()
+        request = requests.Request(options['method'], options['url'], headers=headers).prepare()
+        response = session.send(request)
+        self.stdout.write(response.text)
+        # Log.objects.create(job_id=,statusCode=response.status_code,response=response.text)
```
Comment (jatinkatyal13) [inline cron/management/commands/sendrequest.py]: As discussed we will have job_id as the only parameter.
1. Fetch the Job
2. Make the request

File: services/crontab.py [hunk 2]
```diff
@@ -1,36 +1,42 @@
...
-  def refreshAllJob(self):
-    self.cron.remove_all()
-    self.cron.write()
-    for job in models.Job.objects.all():
-      self.createJob(job)
+    def __init__(self):
+        self.cron = CronTab(user=True)
+
+    def _get_curl_command(self, job):
+        return f'{sys.executable} {os.path.join(settings.BASE_DIR, "manage.py")} sendrequest --url {job.fullURL} --token {job.application.token} --method {job.method} &>> /var/log/poker.logs'
```
Comment (jatinkatyal13) [inline services/crontab.py]: change this command to only send the job_id and don't redirect the output

**Review by jatinkatyal13 — CHANGES_REQUESTED** [review]

File: services/crontab.py (hunk 2 above)
Comment (prateek2211) [reply services/crontab.py]: done

File: cron/management/commands/sendrequest.py (hunk 1 above)
Comment (prateek2211) [reply cron/management/commands/sendrequest.py]: done

**Review by jatinkatyal13 — APPROVED** [review]

**Review by jatinkatyal13 — CHANGES_REQUESTED** [review]
Just one quick change. Please format the code using 2 space indentation.

**Review by jatinkatyal13 — APPROVED** [review]

**Pull request merged**